modules/formulize/include/entriesdisplay.php

```diff
@@ -4575,11 +4575,13 @@ function formulize_gatherDataSet($settings=array(), $searches, $sort="", $order=
             // used for trapping the {BLANK} keywords into their own space so they don't interfere with each other, or other filters
             $addToItsOwnORFilter = false;
 
-            if ("creation_uid" == $key OR "entry_id" == $key) {
-                $ele_type = "text";
-            } else {
-                $elementObject = $element_handler->get($key);
-                $ele_type = $elementObject->getVar('ele_type');
+            $dataHandler = new formulizeDataHandler(false);
```

Comment:
OK, so this needs work, rather than printing out the data, we need to mimic the original behaviour that fixed the error as you had originally. Just we pull the names from the array now and iterate over them, rather than having them hard coded in the if statement. This means that you need to check for the presence of the $key in the array. You can use isset() for that, as in: if(isset($metadataFieldTypes[$key]))....